### PR TITLE
Keymap for keebio/iris rev.4 supporting VIA

### PR DESCRIPTION
**Pull request by yoonbae81**
## Description

New keycap for Keebio.io/iris rev.4.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/yoonbae81/keymap.c [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
```
Comment (zvecr) [inline keyboards/keebio/iris/keymaps/yoonbae81/keymap.c:3]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/yoonbae81/keymap.c [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+#define _MAIN 0
+#define _FN1 1
+#define _FN2 2
```
Comment (zvecr) [inline keyboards/keebio/iris/keymaps/yoonbae81/keymap.c:7]:
```suggestion
enum layer_names {
    _MAIN,
    _FN1,
    _FN2
};
```

File: keyboards/keebio/iris/keymaps/yoonbae81/rules.mk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+VIA_ENABLE = yes
+LINK_TIME_OPTIMIZATION_ENABLE = yes
```
Comment (zvecr) [inline keyboards/keebio/iris/keymaps/yoonbae81/rules.mk:2]:
```suggestion
LTO_ENABLE = yes
ENCODER_ENABLE = no
```

File: keyboards/keebio/iris/rev4/rules.mk [hunk 4]
```diff
@@ -32,4 +32,4 @@ RGBLIGHT_ENABLE = yes       # Enable WS2812 RGB underlight.
 SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
 
 SPLIT_KEYBOARD = yes
-ENCODER_ENABLE = yes
+ENCODER_ENABLE = no
```
Comment (zvecr) [inline keyboards/keebio/iris/rev4/rules.mk:35]:
```suggestion
ENCODER_ENABLE = yes
```

Comment (yoonbae81) [reply keyboards/keebio/iris/rev4/rules.mk]: Thanks, I resolved your all comments.

File: keyboards/keebio/iris/keymaps/yoonbae81/config.h [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#define MOUSEKEY_DELAY 100
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/yoonbae81/config.h:1]:
```suggestion
#pragma once

#define MOUSEKEY_DELAY 100
```

Comment (yoonbae81) [reply keyboards/keebio/iris/keymaps/yoonbae81/config.h:3]: Agreed. I committed again. Thanks.

File: keyboards/keebio/iris/rev4/rules.mk (hunk 4 above)
Comment (yoonbae81) [reply keyboards/keebio/iris/rev4/rules.mk]: I've applied all change requests

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**